### DISCUSSION TITLE
File and image uploader made to be a successor to sharenix

### DISCUSSION THREAD

**Discussion by AlekEagle** [dacap/clip#61] [Show and tell]
Really glad I stumbled across clip because it significantly simplified clipboard interactivity, thanks so much! Using clip I made a file and image uploader that can easily integrate into online services. It stands to be a successor to sharenix, which was recently archived. I use clip to retrieve data from the clipboard to upload and to put links for uploaded resources onto the clipboard. It can be found at https://github.com/AlekEagle/uploadr and is made specifically for Linux systems, but also mostly supports macOS (thanks once again to clip).

**Comment by dacap** [comment]
Hi @AlekEagle, glad to hear that! I'm really happy to see utilities being made with clip, it's just a little library but such an useful feature to interact with the clipboard 🙏  I'll add uploadr to the Wiki page.
